### PR TITLE
Feature/mockup

### PR DESCRIPTION
**Pull request by vetom**
Collaborating for the issue #27

### CODE REVIEW

**Comment by mamodom** [comment]
:eyes:

**Comment by wovalle** [comment]
Hey @vetom, can you post screenshots of the view after the makeover?

**Comment by vetom** [comment]
Here is a preview! @wOvalle 
24c66a6160834e6f8755d2b76c7b86a1

File: src/js/components/createGame.jsx [hunk 1]
```diff
@@ -12,11 +13,26 @@ export default ({
...
 }) => (
   <div>
     <Header name="Home" />
     <div class="col-md-6 col-md-offset-3">
       <div class="panel panel-default">
+        <div class="panel-heading">
+          <div class="row">
+            <div class="col-md-4 pull-right">
+              <div class="pull-right" >
+                {Object.keys(werewolfService.language).map(language => (
```
Comment (wovalle) [inline src/js/components/createGame.jsx:27]: I think we can extract the logic out of the render function

**Comment by wovalle** [comment]
Great job! We need to edit a couple of things and it can be merged!

**Review by mamodom — APPROVED** [review]

**Pull request merged**